R/get_articles.R

```diff
@@ -4,16 +4,14 @@
 #' based on having a preamble and several articles.
 #' Once articles are split,
 #' you can access all the "preambles",
-#' "membership" or "termination" clauses,
+#' "accession" or "termination" clauses,
 #' or articles that contain certain word.
 #' @param textvar A text variable
 #' @param article Would you like to get a specific article?
 #' Null by default.
 #' Other options include the "preamble",
-#' "termination" clause, "membership" clause, "annex",
-#' or an article number.
-#' The specified portion or number
-#' across all treaties will be returned.
+#' "termination" clause, "membership" clause, or "annex".
```

Comment:
Does this maybe need a bit more documentation so that users can follow exactly how these are being selected?